.github/assets/js/command-data.js

```
@@ -0,0 +1,539 @@
+            const content = document.createElement('div');
+            content.style.backgroundColor = 'white';
+            content.style.borderRadius = '8px';
+            content.style.padding = '20px';
+            content.style.maxWidth = '600px';
+            content.style.maxHeight = '80vh';
+            content.style.overflow = 'auto';
+            content.style.boxShadow = '0 4px 20px rgba(0, 0, 0, 0.3)';
+            content.setAttribute('tabindex', '-1'); // Make the content focusable for keyboard events
+            
+            // Media query for dark mode
+            if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)